<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1875</width>
    <height>1183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Параметры</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Интенсивность генерации:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="min_lambda_gen">
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="max_lambda_gen">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>381</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Дисперсия генерации:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="min_sigma_gen">
          <property name="text">
           <string>0.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="max_sigma_gen">
          <property name="text">
           <string>0.3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>331</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Интенсивность обработки:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="min_lambda_proc">
          <property name="text">
           <string>20</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="max_lambda_proc">
          <property name="text">
           <string>70</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>331</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Дисперсия обработки:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="min_sigma_proc">
          <property name="text">
           <string>0.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="max_sigma_proc">
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="calculate">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Вычислить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Произвольная точка факторного пространства</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>x1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x1">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>x2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x2">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>x3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x3">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>x4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x4">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="custom">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Задать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>1851</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Результаты</string>
    </property>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>741</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Уравнение линейной регрессии:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_79">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Уравнение частично нелинейной регрессии:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>31</width>
       <height>39</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>y=</string>
     </property>
    </widget>
    <widget class="QLabel" name="linear_text">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>721</width>
       <height>39</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>31</width>
       <height>39</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>y = </string>
     </property>
    </widget>
    <widget class="QLabel" name="nonlinear_text">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>1801</width>
       <height>39</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>1081</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Матрица планирования</string>
    </property>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1071</width>
       <height>621</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>16</number>
     </property>
     <property name="columnCount">
      <number>21</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
